.github/workflows/unit-tests-pr.yml

```diff
@@ -23,3 +23,10 @@ jobs:
     name: Unit Tests
     if: ${{ github.event.review.state == 'APPROVED' || github.event_name == 'pull_request' }}
     uses: ./.github/workflows/frontend-unit-tests.yml
+
+  load-test:
+    name: Load Tests
+    if: ${{ github.event.review.state == 'APPROVED' || github.event_name == 'pull_request' }}
+    uses: ./.github/workflows/load-test-oha.yml
+    with:
+      oha-version: 'v1.4.5'
```

Comment:
I think this makes more sense to add this in another workflow file, since this one is named "unit tests" and I see `oha` tests more like e2e tests

Reply:
It was in another workflow in the first commit already in [b7b7dce](https://github.com/yeagerai/genlayer-simulator/pull/257/commits/b7b7dced3750c8868fe31089f478e5780f9d7045)
And then I've abstracted these load tests and made it as a reusable workflow now(as a part of unit tests on PR)